### PR TITLE
fixing filtering by home id

### PR DESCRIPTION
**Pull request by gcezar**

### CODE REVIEW

File: portal/app/farm_updater/egauge_api.py [hunk 1]
```diff
@@ -82,7 +83,10 @@ def processing_egauge_data(self):
             print(err)
             return json.dumps(egauge_data)
 
-        queryset = FarmDevice.objects.filter(type=DeviceType.EGAUGE).order_by('-id')[0].device_data
+        # Filtering by home_id and picking the first element in the list
+        home_queryset = FarmDevice.objects.get_queryset().order_by('home_id')
+        queryset = home_queryset.filter(type=DeviceType.EGAUGE)[0].device_data
```
Comment (jongoncalves) [inline portal/app/farm_updater/egauge_api.py]: Change it to:
```
queryset = FarmDevice.objects.filter(type=DeviceType.EGAUGE)
if queryset.count() == 0:
    ...
else:
    data = queryset.order_by('-id')[0].device_data
```

File: portal/app/farm_updater/egauge_api.py [hunk 2]
```diff
@@ -69,6 +69,7 @@ def request_egauge_data(self):
     def processing_egauge_data(self):
         from app.models import FarmDevice
         from app.models import DeviceType
+        from app.models import PowernetUserType
```
Comment (jongoncalves) [inline portal/app/farm_updater/egauge_api.py]: Where are you using PowernetUserType? Seems like we can get rid of it

Comment (gcezar) [reply portal/app/farm_updater/egauge_api.py]: forgot to remove it

File: portal/app/farm_updater/egauge_api.py [hunk 3]
```diff
@@ -84,14 +83,18 @@ def processing_egauge_data(self):
             return json.dumps(egauge_data)
 
         # Filtering by home_id and picking the first element in the list
-        home_queryset = FarmDevice.objects.get_queryset().order_by('home_id')
-        queryset = home_queryset.filter(type=DeviceType.EGAUGE)[0].device_data
+        queryset = FarmDevice.objects.filter(type=DeviceType.EGAUGE)
+        if queryset.count() == 0:
+            print('No egauge device created. Please create one')
+            return None
+        else:
```
Comment (jongoncalves) [inline portal/app/farm_updater/egauge_api.py]: This else is not needed bc of the return above

**Comment by jongoncalves** [comment]
Cool - good to merge.

**Pull request merged**